include/sensor.h

```diff
@@ -143,6 +143,9 @@ enum sensor_channel {
 	/** Current, in amps **/
 	SENSOR_CHAN_CURRENT,
 
+	/** Angular rotation, in degrees */
+	SENSOR_CHAN_ROTATION,
```

Comment:
I wonder if angular position might be better. "Rotation" could mean position or velocity.

Reply:
I thought about it but I think angular position will be understood as absolute value whereas qdec gives relative change (+/- from previous reading).

Reply:
Doesn't that make it velocity then?

Reply:
Velocity is a change of distance in time. Here we have no idea how much time passed only that wheel rotated by `x` angle.